FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <title>FuncGodot Manual</title>
+    <link rel="icon" type="image/x-icon" href="../images/godot_ranger.svg">
+    <link rel="stylesheet" href="../styles.css">
+</head>
+<body>
+    <div class="main">
+        <h1>Runtime map building</h1>
+        <p>Runtime map building is where you build a map in the middle of a running game. This can be used for a number of purposes such as: procedural generation of maps, loading maps from the .map file instead of scenes, and to support user-made custom maps. Runtime map building can also be combined with map editor launch configurations to provide a seamless experience making a map and seeing it in-game.</p>
+        <h2>Setting up Runtime map building in Godot</h2>
+        <p>Runtime map building involves using the method <code>verify_and_build()</code> available under the <a href="ref_func_godot_map.html" target="main">FuncGodotMap</a> node. The idea behind this, is that when you build a map at runtime, you set the <code>map_file</code> field, and then call this method. When this is done, this builds all the entities in the normal way, calling <code>_func_godot_apply_properties(entity_properties: Dictionary)</code> on these entities.</p>
+        <p>When the map builds, it will emit the signals <code>build_complete</code> or <code>build_failed</code>, which you can then connect to, to handle errors gracefully and to run any post-build code you need to run.</p>
+        <p>If you need the UV2 information for baking lightmaps, you can then call FuncGodotMap's <code>unwrap_uv2()</code> method. This generates the UV2 information, which emits the signal <code>unwrap_uv2_complete</code> when it is complete. It is at this point you may bake your Lightmaps.
+        <blockquote><i>As of Godot 4.2, runtime lightmap baking is not currently possible, so instead you should consider using a different lighting solution here.</i></blockquote>
+        An example of runtime map building is as follows:</p>
+        <pre><code>extends Node
+
+@onready var func_godot_map := $FuncGodotMap as FuncGodotMap
+
+# We can connect the various build signals of the FuncGodotMap 
+# in order to add more functionality after building.
+func _ready():
+    func_godot_map.connect("build_complete", _build_complete)
+    func_godot_map.connect("build_failed", _build_failed)
+    func_godot_map.connect("unwrap_uv2_complete", _unwrap_uv2_complete)
+    func_godot_map.verify_and_build()
```

Comment:
Figure we should add the map file line, since we describe that as part of the process earlier.

```
func_godot_map.local_map_file = "res://maps/runtime_building.map"
func_godot_map.verify_and_build()
```